pages/api/team.js

```diff
@@ -0,0 +1,90 @@
+import AirtablePlus from 'airtable-plus'
+
+const airtable = new AirtablePlus({
+    baseID: 'app79wD9AFys1NMUp',
+    apiKey: process.env.AIRTABLE_API_KEY,
+    tableName: 'Current',
+})
+
+const cache = {
+    refreshed_at: 0,
+    current: [],
+    acknowledged: []
+};
+
+export async function fetchTeam() {
+    if ((cache.refreshed_at + (5 * 60 * 1000)) >= Date.now()) {
```

Comment:
Can this be improved?

Reply:
Using `getStaticProps` and `revalidate: <>` now.